### DISCUSSION TITLE
Using a single DB for two MISP instances

### DISCUSSION THREAD

**Discussion by Gh1Dra0** [MISP/MISP#8340] [Q&A]
I want to Implement HA MISP and want to eliminate any DB downtime by distributing the load to two MISP Apps by using HA SQL instances on cloud.

To archive the same is it feasible to use two MISP instances on a single DB.
I have tested it for a while and it doesn't causes any errors but am worried about consistency of the DB.

Any views suggestions?

**Comment by iglocska** [comment]
Yes, it is possible. Here's a sample implementation using AWS: 

https://github.com/0xtf/HAMISPA

the person behind it explaining it:

https://www.youtube.com/watch?v=lUK3otmIcH4


This is just an example, there are plenty of organisations building similar setups, some considerations to take care of:

1. take care of session pinning between your load balanced legs
2. some data is stored on disk, these need to be on a shared resource (config files, malware samples, attachments, organisation logos, etc).
